operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -62,13 +67,13 @@ public static void after() {
         vertx.close();
     }
 
-    @Test
-    public void testSuccessfulReconcile(VertxTestContext context)  {
+
+    public void testSuccessfulReconcile(VertxTestContext context, Labels selectorLabels)  {
```

Comment:
If it is not a test anymore, it might be best to rename it to not start with `test`. Also, the double empty line above is probably not needed?